third_party/move/move-compiler/src/expansion/translate.rs

```diff
@@ -869,24 +900,24 @@ fn attribute_value(
             check_module_name(context, &ident_loc, &mident);
             EV::Module(mident)
         },
-        PV::ModuleAccess(sp!(ident_loc, PN::Two(sp!(aloc, LN::Name(n1)), n2)))
-            if context
-                .named_address_mapping
-                .as_ref()
-                .map(|m| m.contains_key(&n1.value))
-                .unwrap_or(false) =>
-        {
-            let addr = address(context, false, sp(aloc, LN::Name(n1)));
-            let mident = sp(ident_loc, ModuleIdent_::new(addr, ModuleName(n2)));
-            check_module_name(context, &ident_loc, &mident);
-            EV::Module(mident)
+        PV::ModuleAccess(ma) => {
+            let value = match ma {
+                sp!(ident_loc, PN::Two(sp!(aloc, LN::Name(n1)), n2)) => {
+                    // Check to see if `n1::n2` can be resolved to be module access.
+                    can_be_resolved_as_module(context, ident_loc, aloc, n1, n2)
```

Comment:
the original code calls `check_module_name` which further calls `check_for_deprecated_module_use`. Do we need to it here as well?

Reply:
Good catch. I think I accidentally removed it during a final refactoring. Adding it back.